ansible/roles/setup_scan_users/tasks/main.yml

```diff
@@ -0,0 +1,89 @@
+- name: Ceate scanadmin user.
+  user:
+    createhome: yes
+    name: scanadmin
+
+- name: Create scanbasic user.
+  user:
+    createhome: yes
+    name: scanbasic
+
+- name: Setup RHEL7 yum repo.
+  yum_repository:
+    name: "{{ rhel7_yum_repo_name }}"
+    description: RHEL 7 - $basearch
+    file: "{{ yum_repo_file }}"
+    baseurl: "{{ rhel7_yum_repo_url }}"
+    enabled: yes
+    gpgcheck: no
+  when:
+    - ansible_distribution == "RedHat"
+    - ansible_distribution_major_version == "7"
+
+- name: Setup RHEL7 yum extras repo.
+  yum_repository:
+    name: "{{ rhel7_yum_extras_repo_name }}"
+    description: RHEL 7 - $basearch
+    file: "{{ yum_repo_file }}"
+    baseurl: "{{ rhel7_yum_extras_repo_url }}"
+    enabled: yes
+    gpgcheck: no
+  when:
+    - ansible_distribution == "RedHat"
+    - ansible_distribution_major_version == "7"
+
+- name: Setup RHEL6 yum repo.
+  yum_repository:
+    name: "{{ rhel6_yum_repo_name }}"
+    description: RHEL 6 - $basearch
+    file: "{{ yum_repo_file }}"
+    baseurl: "{{ rhel6_yum_repo_url }}"
+    enabled: yes
+    gpgcheck: no
+  when:
+    - ansible_distribution == "RedHat"
+    - ansible_distribution_major_version == "6"
+
+- name: Setup RHEL6 yum extras repo.
+  yum_repository:
+    name: "{{ rhel6_yum_extras_repo_name }}"
+    description: RHEL 6 - $basearch
+    file: "{{ yum_repo_file }}"
+    baseurl: "{{ rhel6_yum_extras_repo_url }}"
+    enabled: yes
+    gpgcheck: no
+  when:
+    - ansible_distribution == "RedHat"
+    - ansible_distribution_major_version == "6"
+
+- name: Add libselinux-python support.
+  yum:
```

Comment:
Will take the opportunity to share this Ansible module http://docs.ansible.com/ansible/latest/package_module.html. It will help if we want to support both yum and dnf.